imap_processing/hi/hi_cdf_attrs.py

```diff
@@ -158,3 +159,54 @@
     var_type="support_data",
     variable_purpose="PRIMARY",
 )
+
+# Histogram attributes
+hi_hist_l1a_global_attrs = GlobalDataLevelAttrs(
+    data_type="L1A_CNT>Level-1A Histogram",
+    logical_source="imap_hi_l1a_hist",
+    logical_source_desc=("IMAP-HI Instrument Level-1A Histogram Data."),
```

Comment:
I think we've been using Hi and Lo rather than all caps for instrument names, but someone else can probably confirm.
```suggestion
    logical_source_desc=("IMAP-Hi Instrument Level-1A Histogram Data."),
```

Reply:
I confirmed that Lo cdf global attributes are not all caps. I will change the other occurrences of "HI" in this file.